common/metrics/config.go

```diff
@@ -399,3 +443,64 @@ func configExcludeTags(cfg ClientConfig) map[string]map[string]struct{} {
 	}
 	return tagsToFilter
 }
+
+func (s SanitizeRange) toTally() (tally.SanitizeRange, error) {
+	startRangeRunes := []rune(s.StartRange)
+	if len(startRangeRunes) != 1 {
+		return tally.SanitizeRange{}, fmt.Errorf("start range '%+v' must be a single rune", startRangeRunes)
+	}
+
+	endRangeRunes := []rune(s.EndRange)
+	if len(endRangeRunes) != 1 {
+		return tally.SanitizeRange{}, fmt.Errorf("end range '%+v' must be a single rune", endRangeRunes)
+	}
+
+	return tally.SanitizeRange([2]rune{startRangeRunes[0], endRangeRunes[0]}), nil
```

Comment:
Do we need the 2 in [2]?

Reply:
yes, that is the tally-specific type. they take in an array of runes, not a slice